pkg/doc/presexch/package_test.go

```diff
@@ -53,3 +59,128 @@ func TestGetContext(t *testing.T) {
 		require.Contains(t, err.Error(), "@context field not found")
 	})
 }
+
+func Test_mDLNestedCtx(t *testing.T) {
+	schemas := []*Schema{{
+		URI: "https://example.org/examples#mDL",
+	}}
+
+	creds := []*verifiable.Credential{
+		{
+			Context: []string{
+				verifiable.ContextURI,
+				"https://trustbloc.github.io/context/vc/examples/mdl-v1.jsonld",
+			},
+			Types: []string{verifiable.VCType, "mDL"},
+		},
+	}
+
+	t.Run("success", func(t *testing.T) {
+		mDLjson := `{
```

Comment:
test files should be created as a file in a testdata subfolder and then accessed using embed.

example:
https://github.com/hyperledger/aries-framework-go/blob/af30e870c3c085ed1debfe304de293ba2b995516/pkg/doc/signature/jsonld/processor_test.go#L509

Reply:
fixed